warehouse/models/mart/gtfs_schedule_latest/dim_translations_latest.sql

```diff
@@ -0,0 +1,9 @@
+WITH
+dim_translations_latest AS (
```

Comment:
AFAIK this is empty, have we checked how this shows up on the open data site? Just curious

Reply:
This luckily doesn't break anything; CKAN handles emptiness gracefully - it shows up with columns fully defined, and spits out a headings-only CSV when downloaded.